### DISCUSSION TITLE
Tizen 2.3 (2015) support?

### DISCUSSION THREAD

**Discussion by banai722** [jhassan8/crunchyroll-tizen#103] [Q&A]
Hello, I have a UN32J5500 tv updated to the latest version (1550), is it possible to install the app on it? If it´s possible, how can I do it?
Pd: Sorry if my english is not very well :(

**Comment by jhassan8** [comment]
It should work, I just don't know if the way to install the app is the same, I would have to try.

**Reply by Sidewinder-2011** [reply]
Hiya was you able to get it work on tizen 2.3 ? When I try and install it , it just a boots to a  black screen. On my other tv with 7.0 work fine
